src/comp_loinc/loinc_builder_steps.py

```diff
@@ -805,25 +794,30 @@ def save_to_owl(
     ):
         """Save the current module to an OWL file."""
         logger.info(f"save-owl: Starting")
-        owl_dumper = OWLDumper()
-        document = owl_dumper.to_ontology_document(
-            schema=self.runtime.current_schema_view.schema,
-            element=list(self.runtime.current_module.get_all_entities()),
-        )
-        document.ontology.iri = funowl.identifiers.IRI(
-            f"https://comploinc/{self.runtime.current_module.name}"
-        )
         owl_file_path = file_path
         if owl_file_path is None:
             owl_file_path = Path(self.runtime.current_module.name + ".owl")
         if not owl_file_path.is_absolute():
             outdir = self.configuration.output
             if self.configuration.fast_run:
                 outdir = outdir / "fast-run"
-            owl_file_path = outdir / owl_file_path
+        owl_file_path = outdir / owl_file_path
 
         owl_file_path.parent.mkdir(parents=True, exist_ok=True)
         typer.echo(f"Writing file: {owl_file_path}")
-        with open(owl_file_path, "w") as f:
-            f.write(str(document))
-        logger.info(f"save-owl: Done")
+
+        owl_serializer = OwlSerializer(self.runtime, owl_file_path)
+        owl_serializer.as_tbox()
```

Comment:
I see this is the primary change of the PR, changing from `OWLDumper`.

I'm guessing `.as_tbox()` does the writing?